### PR TITLE
Extensible enumerations support

### PR DESCRIPTION
**Pull request by paulwarren-wk**
This PR adds support for extensible enumeration facts.

Previously, such facts showed the raw value in the fact inspector, e.g. `http://fasb.org/us-gaap/2021-01-31#AccruedLiabilitiesMember`

These facts now show the standard label for the member(s) in the enumeration value, e.g.:

image

Fixes XT-1334

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: iXBRLViewerPlugin/__init__.py
Comment (brettkail-wk) [inline iXBRLViewerPlugin/__init__.py:44]: Remove unused?

File: iXBRLViewerPlugin/iXBRLViewer.py
Comment (brettkail-wk) [inline iXBRLViewerPlugin/iXBRLViewer.py:233]: What does this do if the fact value is invalid?  Do we need some kind of `qnEnums is not None`?

File: iXBRLViewerPlugin/viewer/src/js/fact.js
Comment (brettkail-wk) [inline iXBRLViewerPlugin/viewer/src/js/fact.js:106]: Should we wait on the other PR and `getLabelOrName` here?

File: iXBRLViewerPlugin/viewer/src/js/fact.js
Comment (brettkail-wk) [inline iXBRLViewerPlugin/viewer/src/js/fact.js:108]: Do you think people would prefer to see this as a bulleted list rather than a comma-separated list?  If we don't yet have enough experience with enum set, I'm fine with this for now.

**Review by brettkail-wk — APPROVED** [review]

**Review by sagesmith-wf — APPROVED** [review]

File: iXBRLViewerPlugin/__init__.py
Comment (paulwarren-wk) [reply iXBRLViewerPlugin/__init__.py:44]: They are used (see [line 74](https://github.com/Workiva/ixbrl-viewer/pull/228/files/1751ec0fcef4eabed6e1d81bc0a25a40fb3e3cb2#diff-a65816b2086d1b8e75aabec4dd3761c8795d7693a1d186400b4718c1a894fceaR74))

I managed to trigger an exception during development which highlighted that these imports were missing.

Comment (brettkail-wk) [reply iXBRLViewerPlugin/__init__.py:44]: Ah, thanks, I overlooked the commit message, sorry.

File: iXBRLViewerPlugin/viewer/src/js/fact.js
Comment (paulwarren-wk) [reply iXBRLViewerPlugin/viewer/src/js/fact.js:110]: Yes, good point.  I've now merged the other branch into this one and made that change, so this PR is currently showing the changes in both, but it should go back to just this once #224 is merged.

File: iXBRLViewerPlugin/viewer/src/js/fact.js
Comment (paulwarren-wk) [reply iXBRLViewerPlugin/viewer/src/js/fact.js:112]: Just had a quick chat with Dave Shaw on use in the US-GAAP taxonomy, and it sounds like most cases will actually be a single item, but where is more than one, a bulleted list would be clearer, so I'll make that change.

**Review by sagesmith-wf — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**
